app/channels/editor_channel.rb

```diff
@@ -0,0 +1,59 @@
+# The Editor Channel is opened when a user navigates to the `Edit` page of an
+# Evidence, Issue, Note, or Card. The channel is closed when the user navigates
```

Comment:
Are we closing the channel or simply unsubscribing the user? The difference can have an implication to how we would handle conflict management

Reply:
> The difference can have an implication to how we would handle conflict management
Can you explain. I'm not sure how it would make a difference.

I'd say it's actually just unsubscribing them though. The channel would technically still be available for someone else to subscribe to.

Note: a subscription to this channel can't be "shared". IE two users who subscribed to the same channel, with the same resource will not receive eachother updates. They'll both be updating the same record yes, but the broadcasting will always be specific to the user who performed an action.
IE if you and I were both updating the same issue.
I make a change
 - the issue updates
 - the channel broadcasts the updated time
 - only I receive the broadcast
Your editor does not know that a change has occured.

We do no real time conflict management. Conflict resolution happens exactly as it does now. When a user saves, they will see a conflict warning.

Reply:
Thanks for the explanation! My original thought was: if we're not closing the channel, we can broadcast an error/alert message to other users subscribed to the channel that a changed as been made. Though, it seems we're kinda doing something similar with the `updated_at` time in your other comment. 👍

Reply:
We definitely have an opportunity to broadcast conflicts in real time. Could be a good enhancement if we have time.